### ISSUE TITLE
Add the usual social bookmarking tools

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-legacy#202]
We should have the twitter, g+, etc buttons somewhere so people can talk about their fav disease etc.

I like how NIF has this on their pages now, in a not too prominent but easy to find location on the righthand side.

Probably don't want this until July release?

**Comment by sarahjkim** [comment]
Just making a note to self:
Facebook: https://developers.facebook.com/docs/plugins/like-button
Google: https://developers.google.com/+/web/+1button/
Twitter: https://dev.twitter.com/docs/tweet-button

**Comment by jmcmurry** [comment]
Agreed this would be great, but it is also more involved than it would first appear. It depends on persistence of URLs at the entity level and also whether we implement rich snippets that are embeddable the way that folks have come to expect.

**Comment by kltm** [comment]
Just as a thought: if this is being used for medical-type purposes, what are the implications of adding external tracking beacons designed to individually identify users throughout the pages?

**Comment by jmcmurry** [comment]
Our aims are to enable users to:
A. Spread the news of our existence using social media
B. Follow us on FB or Twitter
C. Share information about their favorite thing [phenotype, disease, gene etc] in the same way that a reader might share a newspaper article. 

We could provide the social media buttons:
1. On the homepage only 
2. At the level of each resource page (eg. NYT, Guardian, etc)
3. Both 1 & 2 (eg. NIF, WebMD)

In all three cases, although we are concerned with tracking _our_ aggregate impact, our goal isn't to track a specific user or his/her impact, though others theoretically could. Is this your concern, @kltm? Is the presence of the buttons alone enough to make people uneasy? 

For now, I would propose the scope of the ticket is just A and B and thus homepage buttons only. A separate ticket can be logged for the longer range goal of page-level sharing and citation as this requires more thought and engineering.

**Comment by kltm** [comment]
While the people of monarch might not be tracking individual people, the external code that you're running that produces those buttons does, even if the buttons are not interacted with, and that information is individualized, gathered, and sold on. Thus funds the current iteration of the big sites. Having the share buttons (same thing as trackers, just different ends of same thing) on the homepage would only allow site-level tracking (e.g. this user went to monarch at this time for approximately this duration); page-level sharing gives upstream tracking very specific view of an individual moving around a site and what their interests are.

To reiterate, I'm not taking a particular stand on this, but I was wondering if consideration had been given that the intended audience of some of monarch might (or might not) be sensitive to this kind of thing, especially as some things are maybe less research-y and more medical-y.

Also, these comments are specific to "social" sharing via the major information brokers, and not necessarily citation mechanisms (zotero interaction, etc.).

**Comment by jmcmurry** [comment]
Thank you for clarifying. I didn't realize that the location of the buttons within a site determined the granularity of the data that the brokers collect about users, but it makes perfect sense. All food for thought and yet another reason to hold off of page-level social buttons for now. Site level should be OK; there's nothing particularly revealing about having visited the homepage.
